reporters/reporter.go

```diff
@@ -52,15 +51,11 @@ func copyOpts(src map[string]interface{}) map[string]interface{} {
 
 // Report is Reporters' implementation of the Reporter interface
 func (r *Reporters) Report(event string, opts map[string]interface{}) error {
```

Comment:
To clarify, this function is used in many places in the project. Especially [here](https://github.com/topfreegames/maestro/blob/v9/models/room.go#L333).